01-shapefile-attributes.Rmd

```diff
@@ -133,19 +133,22 @@ extent(x = point)
 
```

Comment:
gotcha -- this looks like a merge conflict to me. But i'm not sure where it happened. we may have lost a bit of code but we can figure it out!